fili-sql/src/test/groovy/com/yahoo/bard/webservice/sql/builders/SimpleDruidQueryBuilder.groovy

```diff
@@ -0,0 +1,229 @@
+import static com.yahoo.bard.webservice.database.Database.REGION_ISO_CODE
+import static com.yahoo.bard.webservice.database.Database.REGION_NAME
+import static com.yahoo.bard.webservice.database.Database.USER
+import static com.yahoo.bard.webservice.database.Database.WIKITICKER
+import static java.util.Arrays.asList
+
+import com.yahoo.bard.webservice.data.config.dimension.DefaultDimensionField
+import com.yahoo.bard.webservice.data.config.dimension.DefaultKeyValueStoreDimensionConfig
+import com.yahoo.bard.webservice.data.config.names.DataSourceName
+import com.yahoo.bard.webservice.data.config.names.TableName
+import com.yahoo.bard.webservice.data.dimension.Dimension
+import com.yahoo.bard.webservice.data.dimension.MapStoreManager
+import com.yahoo.bard.webservice.data.dimension.impl.KeyValueStoreDimension
+import com.yahoo.bard.webservice.data.dimension.impl.ScanSearchProviderManager
+import com.yahoo.bard.webservice.data.time.DefaultTimeGrain
+import com.yahoo.bard.webservice.data.time.ZonedTimeGrain
+import com.yahoo.bard.webservice.data.time.ZonelessTimeGrain
+import com.yahoo.bard.webservice.druid.model.aggregation.Aggregation
+import com.yahoo.bard.webservice.druid.model.datasource.TableDataSource
+import com.yahoo.bard.webservice.druid.model.filter.Filter
+import com.yahoo.bard.webservice.druid.model.having.Having
+import com.yahoo.bard.webservice.druid.model.orderby.LimitSpec
+import com.yahoo.bard.webservice.druid.model.postaggregation.PostAggregation
+import com.yahoo.bard.webservice.druid.model.query.Granularity
+import com.yahoo.bard.webservice.druid.model.query.GroupByQuery
+import com.yahoo.bard.webservice.druid.model.query.TimeSeriesQuery
+import com.yahoo.bard.webservice.metadata.DataSourceMetadata
+import com.yahoo.bard.webservice.metadata.DataSourceMetadataService
+import com.yahoo.bard.webservice.table.Column
+import com.yahoo.bard.webservice.table.ConfigPhysicalTable
+import com.yahoo.bard.webservice.table.ConstrainedTable
+import com.yahoo.bard.webservice.table.PhysicalTableDictionary
+import com.yahoo.bard.webservice.table.StrictPhysicalTable
+import com.yahoo.bard.webservice.table.resolver.DataSourceConstraint
+import com.yahoo.bard.webservice.util.Utils
+
+import org.joda.time.DateTimeZone
+import org.joda.time.Interval
+
+import java.util.stream.Collectors
+
+class SimpleDruidQueryBuilder {
+    public static final String START = "2015-09-12T00:00:00.000Z"
+    public static final String END = "2015-09-13T00:00:00.000Z"
+
+    public static PhysicalTableDictionary getDictionary() {
+        return getDictionary("", "")
+    }
+
+    public static PhysicalTableDictionary getDictionary(String apiPrepend, String fieldPrepend) {
+        def dataSource = getWikitickerDatasource(apiPrepend, fieldPrepend)
+        PhysicalTableDictionary physicalTableDictionary = new PhysicalTableDictionary()
+        physicalTableDictionary.put(WIKITICKER, dataSource.getPhysicalTable().sourceTable as ConfigPhysicalTable)
```

Comment:
`dataSource.physicalTable` is syntactic sugar for `dataSource.getPhysicalTable()`